src/communication/network_interface_cl.c

```diff
@@ -11541,3 +11541,106 @@ tdes_reset_query_start_info (PT_NODE * node)
     }
 #endif
 }
+
+/* lob_create_dir - create external lob directory
+ *
+ * return:
+ *
+ *   hfid(in): When creating the LOB directory, use each table's HFID as the directory name to distinguish them
+ *   attrid_arr (in): An array that stores LOB attribute ids of the table.
+                      When creating the LOB directory, each LOB attribute is distinguished by its id
+ *   lob_attrid_arr_length(in): length of attrid_arr
+ */
+int
+lob_create_dir (HFID * hfid, int *attrid_arr, int lob_attrid_arr_length)
+{
+#if defined(CS_MODE)
+  int error = ER_NET_CLIENT_DATA_RECEIVE;
+  int req_error, request_size;
+  char *ptr;
+  char *request, *reply;
+  OR_ALIGNED_BUF (OR_INT_SIZE) a_reply;
+
+  reply = OR_ALIGNED_BUF_START (a_reply);
+
+  request_size = OR_HFID_SIZE + OR_INT_SIZE + (OR_INT_SIZE * lob_attrid_arr_length);
+  request = (char *) malloc (request_size);
+  if (request == NULL)
+    {
+      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_OUT_OF_VIRTUAL_MEMORY, 1, (size_t) request_size);
+      return ER_OUT_OF_VIRTUAL_MEMORY;
+    }
+
+  ptr = or_pack_hfid (request, hfid);
+  ptr = or_pack_int_array (ptr, lob_attrid_arr_length, attrid_arr);
+
+  req_error =
+    net_client_request (NET_SERVER_LOB_CREATE_DIR, request, request_size, reply,
+			OR_ALIGNED_BUF_SIZE (a_reply), NULL, 0, NULL, 0);
+  if (req_error == NO_ERROR)
+    {
+      ptr = or_unpack_errcode (reply, &error);
+    }
+
+  if (request)
+    {
+      free_and_init (request);
+    }
```

Comment:
free_and_init 매크로에서 request에 대한 NULL 검사를 수행합니다.
```suggestion
  free_and_init (request);
```

Reply:
반영했습니다.